src/es5processor.ts

```diff
@@ -270,6 +270,7 @@ class ES5Processor extends Rewriter {
       modName = nsImport;
       isNamespaceImport = true;
     } else {
+      // TODO(martinprobst): why doesn't this need stripping "rootDir"?
```

Comment:
this.file.fileName is a TS object, which should have an absolute path.  But pathToModuleName docs say it expects a relative path?

Reply:
Yeah, that's what I'm asking myself. But in tests, this seems to get the right, relative path.

Reply:
In the past when this has happened, it's because we fed in a bad path into one of the topmost TS APIs (like the one that parses tsconfig.json).  "Bad" here is a relative path, because TS works with absolute paths.  The symptom is that most things mostly still work up until you get some really crazy path stuff where some files are absolute and others aren't.

It's probably not worth tracking down in this CL.